src/patternfly/layouts/Flex/examples/Flex.css

```diff
@@ -1,6 +1,5 @@
-.ws-core-l-flex-border,
-.ws-core-l-flex .pf-l-flex .pf-l-flex,
-.ws-core-l-flex .pf-l-flex__item {
```

Comment:
These styles shouldn't be in this PR.

Reply:
Looks like this change got pulled in when rebasing, I've reverted these changes so this file & any changes are no longer part of the PR.